mailbox/api/src/main/java/org/apache/james/mailbox/MessageIdManager.java

```diff
@@ -39,4 +42,6 @@
     void delete(MessageId messageId, List<MailboxId> mailboxIds, MailboxSession mailboxSession) throws MailboxException;
 
     void setInMailboxes(MessageId messageId, List<MailboxId> mailboxIds, MailboxSession mailboxSession) throws MailboxException;
+
+    MessageMetaData appendMessage(InputStream msgIn, Date internalDate, Flags flagsToBeSet, MailboxId mailboxId, MailboxSession mailboxSession) throws MailboxException;
```

Comment:
I don't understand what's the point of that : you already have this exact feature in other managers